docs/src/main/asciidoc/writing-native-applications-tips.adoc

```diff
@@ -330,6 +330,17 @@ and in the case of using the Maven configuration instead of `application.propert
 ----
 ====
 
+=== Working with Random and SplittableRandom
+
+If your native application is initialized at build time and has static `java.util.Random` or `java.util.SplittableRandom` instances then its native build will fail because embedding such instances in native image will result in the same cached seed value be reused which will negatively affect the entropy (randomness) quality of the produced values.
+
+There is a number of techniques that can be used to have static `Random` and `SplittableRandom` instances securely used in native images:
+ * Do the runtime initialization instead of the build-time initialization
+ * Keep the build-time initialization but request a runtime re-inititalization of the affected classes
+ * Substitute instances of `Random` and `SplittableRandom` with `null`
```

Comment:
How is it a workable option? The chances that they are not actually used are extremely narrow.

Reply:
@gsmet For example, there were `Random` instances used from a test algorithm code in BC FIPS case, https://github.com/quarkusio/quarkus/blob/main/extensions/security/runtime/src/main/java/io/quarkus/security/runtime/graal/BouncyCastleSubstitutions.java#L24, cases like that...